docs/reference/mapping/params/analyzer.asciidoc

```diff
@@ -1,81 +1,17 @@
 [[analyzer]]
 === `analyzer`
 
-The values of <<mapping-index,`analyzed`>> string fields are passed through an
-<<analysis,analyzer>> to convert the string into a stream of _tokens_ or
-_terms_.  For instance, the string `"The quick Brown Foxes."` may, depending
-on which analyzer is used,  be analyzed to the tokens: `quick`, `brown`,
-`fox`.  These are the actual terms that are indexed for the field, which makes
-it possible to search efficiently for individual words _within_  big blobs of
-text.
+[IMPORTANT]
+====
+Only <<text,`text`>> fields support the `analyzer` mapping parameter.
+====
 
-This analysis process needs to happen not just at index time, but also at
-query time: the query string needs to be passed through the same (or a
-similar) analyzer so that the terms that it tries to find are in the same
-format as those that exist in the index.
```

Comment:
Covered in [index and search analysis][0].

[0]:https://www.elastic.co/guide/en/elasticsearch/reference/master/analysis-index-search-time.html

Reply:
Maybe it would it make sense to include some small text snippet that links to this broader overview article? I always find it useful to be able to click through from some small, concise parameter description to the more general explanation of the concepts, I might miss them otherwise.

Reply:
There is a link to this section in line 12:

```
Unless overridden with the <<search-analyzer,`search_analyzer`>> mapping
parameter, this analyzer is used for both <<analysis-index-search-time,index and
search analysis>>. See <<specify-analyzer>>.
```